samples/nrf_rpc/protocols_serialization/ps_server/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(protocols_serialization_server)
+
+# NORDIC SDK APP START
```

Comment:
Unnecessary comments wrapping out.

Reply:
This comment Is actually in all of our cmake samples files. What you don't like about it?

Reply:
Ok, probably it requires general clean up.